pkg/restore/restore.go

```diff
@@ -814,6 +823,36 @@ func hasControllerOwner(refs []metav1.OwnerReference) bool {
 	return false
 }
 
+// isCompleted returns whether or not an object is considered completed
+// Used to identify whether or not an object should be restored. Only Jobs or Pods are considered
+func isCompleted(obj *unstructured.Unstructured, groupResource schema.GroupResource) (bool, error) {
+	// TODO(0.9): These are copied from backup/item_backupper. Definititions should be moved
+	// to a shared package and imported here and in the backup package.
+	podsGroupResource := schema.GroupResource{Group: "", Resource: "pods"}
```

Comment:
I'd make these package-level `var`s (just above this function) so they aren't allocated over and over again